assets/scripts/build_gallery.py

```diff
@@ -320,10 +327,46 @@ def get_expectation_instances(expectations_info):
                 f"\n\n----------------\n{expectation_name} ({expectations_info[expectation_name]['package']})\n"
             )
             expectation_tracebacks.write(traceback.format_exc())
-            continue
+        except pydantic.ValidationError:
+            expectation_tracebacks.write(traceback.format_exc())
+        except IndexError:
+            expectation_tracebacks.write(
+                f"Expectation {expectation_name} has invalid test case."
+            )
+            expectation_tracebacks.write(traceback.format_exc())
+        except Exception as exc:
+            expectation_tracebacks.write(f"Unexpected error occurred: {exc}")
+            expectation_tracebacks.write(traceback.format_exc())
     return expectation_instances
 
 
+def create_expectation_instance(expectation_class: type[Expectation]) -> Expectation:
+    """Accepts an Expectation class and attempts to instantiate it with valid parameters
+    obtained from the Expectation's test cases.
+    """
+    expectation_params: dict
+    if expectation_class.examples:
+        # take the first test case available:
+        expectation_params = expectation_class.examples[0]["tests"][0]["in"]
```

Comment:
Are we confident these always work? We have some test cases that are failure cases (at least in the JSON files).

Reply:
We don't need these to be success cases - as long as they have the correct arguments of the correct type, it works enough to instantiate the expectation. As far as I know these test cases aren't supposed to pass in unexpected parameters. There might be a couple wayward examples, and i think we can clean them up.